src/NUnitFramework/framework/Internal/Execution/EventPump.cs

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
 
+using NUnit.Framework.Interfaces;
```

Comment:
Do we use the System namespace first or pure alphabetical convention in NUnit? I thought it was the first.

Reply:
Didnt even notice,  good point. Fixed.